### PR TITLE
The framework should expose its own 'merge' and app-module-path' modules

### PR DESCRIPTION
**Pull request by racodond**

### CODE REVIEW

File: template/wdio.conf.js [hunk 1]
```diff
@@ -50,4 +48,4 @@ const config = {
 
 };
 
-exports.config = merge.recursive(true, require('ntaf/lib/conf/wdio.conf').config, config);
+exports.config = require('ntaf').merge.recursive(true, require('ntaf/lib/conf/wdio.conf').config, config);
```
Comment (alcaneo) [inline template/wdio.conf.js]: I would prefer to declare it in a variable and use the variable here.
I think it is not readable enough.

File: template/wdio.debug.conf.js [hunk 2]
```diff
@@ -12,4 +10,4 @@ const config = {
   execArgv: ['--expose_debug_as=v8debug'],
 };
 
-exports.config = merge.recursive(true, require('./wdio.local.conf').config, config);
+exports.config = require('ntaf').merge.recursive(true, require('./wdio.local.conf').config, config);
```
Comment (alcaneo) [inline template/wdio.debug.conf.js]: Same as above

File: template/wdio.local.conf.js [hunk 3]
```diff
@@ -47,4 +45,4 @@ const config = {
 
 };
 
-exports.config = merge.recursive(true, require('./wdio.conf').config, config);
+exports.config = require('ntaf').merge.recursive(true, require('./wdio.conf').config, config);
```
Comment (alcaneo) [inline template/wdio.local.conf.js]: Same :-)

**Comment by racodond** [comment]
@alcaneo: Fixed!

**Pull request merged**